sdk/go/http/ofrep.sdk.gen.go

```diff
@@ -0,0 +1,49 @@
+// Code generated by protoc-gen-go-flipt-sdk. DO NOT EDIT.
+
+package http
+
+import (
+	context "context"
+	ofrep "go.flipt.io/flipt/rpc/flipt/ofrep"
+	grpc "google.golang.org/grpc"
+	protojson "google.golang.org/protobuf/encoding/protojson"
+	io "io"
+	http "net/http"
+	url "net/url"
+)
+
+type OFREPServiceClient struct {
+	client *http.Client
+	addr   string
+}
+
+func (x *OFREPServiceClient) GetProviderConfiguration(ctx context.Context, v *ofrep.GetProviderConfigurationRequest, _ ...grpc.CallOption) (*ofrep.GetProviderConfigurationResponse, error) {
+	var body io.Reader
+	var values url.Values
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, x.addr+"/ofrep/*/ofrep/v1/configuration", body)
```

Comment:
This is the generated path I was referring to. Could either make it a real namespace parameter or we could skip this generation altogether.
https://github.com/flipt-io/flipt/blob/50af0bc4d6f6e94a58bdfd0108703e82095acd15/rpc/flipt/evaluation/evaluation.proto#L201-L204

Reply:
Sure, I'll put the ignore comment and by the @markphelps message I guess we can just get ride of `namespace` thing